charts/newrelic-k8s-metrics-adapter/README.md

```diff
@@ -9,6 +9,7 @@ A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 ## Source Code
 
 * <https://github.com/newrelic/newrelic-k8s-metrics-adapter>
+* <https://github.com/newrelic/newrelic-k8s-metrics-adapter/tree/main/charts/newrelic-k8s-metrics-adapter>
```

Comment:
This file is generated automatically from the README template, adding this to `sources` in the `Chart.yaml` and running `helm-docs` is the preferred way to add it here :)

Reply:
Thanks for the suggestion. Done in 612a304044e4a507628ec1453a3aacfecd3e5191.